### PR TITLE
Update docu fcc

### PR DESCRIPTION
**Pull request by forsthug**

### CODE REVIEW

File: doc/appendix-de-kassensichv/operation-modes/scu/deutsche-fiskal.md [hunk 1]
```diff
@@ -33,6 +34,10 @@ The Deutsche Fiskal Cloud-SCU is limited to a single client registration (only o
 
 The _fiskaltrust.Middleware.SCU.DE.DeutscheFiskal_-package connects the middleware with a Deutsche Fiskal Cloud-TSE via the _Fiskal Cloud Connector_ ("FCC") service provided by Deutsche Fiskal. The FCC can be installed automatically when the SCU package is started for the first time, or an already installed version of the FCC can be used (see _Parameter_).
 
+**from version:** 1.3.23
+
+The _fiskaltrust.Middleware.SCU.DE.SwissbitCloud_-package handels updates for FCC on start and new versions of the FCC are installed automatically.
```
Comment (TSchmiedlechner) [inline doc/appendix-de-kassensichv/operation-modes/scu/deutsche-fiskal.md:39]:
```suggestion
The _fiskaltrust.Middleware.SCU.DE.SwissbitCloud_ package handles updates for FCC on start and new versions of the FCC are installed automatically when updating the SCU.
```

File: doc/appendix-de-kassensichv/operation-modes/scu/swissbit-cloud.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ The _fiskaltrust.Middleware.SCU.DE.SwissbitCloud_-package connects the middlewar
 
 **from version:** 1.3.23
 
-The _fiskaltrust.Middleware.SCU.DE.SwissbitCloud_-package handels updates for FCC on start and new versions of the FCC are installed automatically.
+The _fiskaltrust.Middleware.SCU.DE.SwissbitCloud_-package handles updates for FCC on start and new versions of the FCC are installed automatically when updating the SCU.
```
Comment (AxelKutschera) [inline doc/appendix-de-kassensichv/operation-modes/scu/swissbit-cloud.md:39]:
```suggestion
The _fiskaltrust.Middleware.SCU.DE.SwissbitCloud_-package handles updates for FCC on start and new versions of the FCC are installed automatically when updating the SCU and rebuild the configuration of the CashBox.
```

File: doc/appendix-de-kassensichv/operation-modes/scu/deutsche-fiskal.md [hunk 3]
```diff
@@ -36,7 +36,7 @@ The _fiskaltrust.Middleware.SCU.DE.DeutscheFiskal_-package connects the middlewa
 
 **from version:** 1.3.23
 
-The _fiskaltrust.Middleware.SCU.DE.SwissbitCloud_ package handles updates for FCC on start and new versions of the FCC are installed automatically when updating the SCU.
+The _fiskaltrust.Middleware.SCU.DE.DeutscheFiskal_-package package handles updates for FCC on start and new versions of the FCC are installed automatically when updating the SCU.
```
Comment (AxelKutschera) [inline doc/appendix-de-kassensichv/operation-modes/scu/deutsche-fiskal.md:39]:
```suggestion
The _fiskaltrust.Middleware.SCU.DE.DeutscheFiskal_-package package handles updates for FCC on start and new versions of the FCC are installed automatically when updating the SCU and rebuild the configuration of the CashBox.
```

**Review by AxelKutschera — COMMENTED** [review]
Does this addition make sense?

**Review by volllly — APPROVED** [review]

**Pull request merged**